webpack/webpack.config.dev.js

```diff
@@ -85,7 +85,7 @@ export default {
           }, {
             loader: 'sass-loader',
             options: {
-              includePaths: [path.resolve(__dirname, '../src', 'scss')],
+              includePaths: [path.resolve(__dirname, '../src', 'scss'), './node_modules'],
```

Comment:
Why is that you're adding node_modules to sass compiling?

Reply:
So the sass compiler can access the Drawer's css included in `src/styles/styles.scss` line 2